task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskFormDatePickers.vue

```diff
@@ -247,7 +247,12 @@ export default {
         return new Date(date);
       } else if (typeof date === 'object') {
         const formattedDate = `${date.year + 1900}-${date.month + 1}-${date.date}`;
-        return new Date(formattedDate);
+        const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+        if (isSafari){
+          return new Date(formattedDate.replace(/-/g, '/'));
+        } else {
+          return new Date(formattedDate);
+        }
```

Comment:
if the format `yyyy/mm/dd` is recognized by all browsers, it would be better IMO to adopt it everywhere without test on userAgent.
```suggestion
        const formattedDate = `${date.year + 1900}/${date.month + 1}/${date.date}`;
        return new Date(formattedDate);
```